### PR TITLE
[DEV-4803] Support repositories in GraphQL metricCurrentValues query

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: server/athenian/api/align/queries/metrics.py [hunk 1]
```diff
@@ -80,8 +80,11 @@ async def resolve_metrics_current_values(
     )
     time_interval = _parse_time_interval(params)
     teams_flat = flatten_teams(team_rows)
+    if (repositories := params.get(MetricParamsFields.repositories)) is not None:
+        repositories = tuple(RepositoryName.from_prefixed(r).unprefixed for r in repositories)
```
Comment (vmarkovtsev) [inline server/athenian/api/align/queries/metrics.py:84]: We must check the repository access using `AccessChecker`. `acc_id` is our final line of defense, we shouldn't overrely on it.

**Review by vmarkovtsev — COMMENTED** [review]
Other than the access check, LGTM!

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]
Wrong radio button

File: server/athenian/api/align/queries/metrics.py (hunk 1 above)
Comment (gaetano-guerriero) [reply server/athenian/api/align/queries/metrics.py]: added

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**